jep-template/0000/README.adoc

```diff
@@ -0,0 +1,176 @@
+= JEP-0000: :bulb: Title :bulb:
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.**JEP Template**
+[TIP]
+====
+In this document, all text in a "Tip" block (or inline text with with a ":bulb:" on either side)
+MUST be remove and/or replaced with text appropriate to this JEP before submission.
+
+Sections may include aditional help and advice in comments.
+":bulb:" entries in comments only need to be filled in if that text is uncommented.
+Comm
+
+See JEP-1 for full and accurate description of the JEP process and what is required in each section.
```

Comment:
Optimistically hyperlink to https://github.com/jenkinsci/jep/blob/master/jep/1/README.adoc ? (Or what is the expected URL permalink format for JEPs?)